nmrfx-analyst-gui/src/main/java/org/nmrfx/analyst/gui/FileMenuActions.java

```diff
@@ -47,6 +49,8 @@ public void basic() {
         menu.getItems().addAll(openMenuItem, openDatasetMenuItem,
                 recentFIDMenuItem, recentDatasetMenuItem, datasetBrowserMenuItem,
                 graphicsMenu);
+
+        PluginLoader.getInstance().registerPluginsOnEntryPoint(EntryPoint.MENU_FILE, menu);
```

Comment:
The other option was in makeMenuBar() in AnalystApp, but this works too.

Reply:
moved to makeMenuBar